### PR TITLE
Readme overhaul

### PR DESCRIPTION
**Pull request by Shahil-Ayato**
Our current Readme.md file at the moment looks ok but a bit outdated and has some redundancy in it. This overhauls the readme file to look much better and cleaner.

*Changes*
- a bigger and higher resolution logo now in the center instead of on the side
- pronunciation.
- more clarity on the project opening paragraph.
- made the building instructions more clear.
- grammar fixes.
- removed the help heading and merged it with contact.
- added a donation heading with a link to our Patreon if anyone wishes to support the project financially with notes on how it would be spent.
- added links to the licenses.
- added LibHac in the credits.

### CODE REVIEW

**Review by Thog — APPROVED** [review]

**Review by AcK77 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
-## Contact
+<a href="https://www.patreon.com/ryujinx">
+    <img src="https://images.squarespace-cdn.com/content/v1/560c1d39e4b0b4fae0c9cf2a/1567548955044-WVD994WZP76EWF15T0L3/Patreon+Button.png?format=500w" width="150">
+</a>
 
-If you have contributions, need support, have suggestions, or just want to get in touch with the team, join our [Discord server](https://discord.gg/Ryujinx)!
-
-If you'd like to donate, please take a look at our [Patreon](https://www.patreon.com/ryujinx).
+All the developers working on the project do so on their free time, but the project has several expenses:
+* Hackable Nintendo Switch consoles to reverse-engineer the hardware and its components
```
Comment (gdkchan) [inline README.md:125]: `and its components` is redundant.

File: README.md [hunk 2]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
+* Hardware to test and fix any bugs certain hardware may have
```
Comment (gdkchan) [inline README.md:126]: Hardware was already listed.

File: README.md [hunk 3]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
+* Software licenses to help developers use whatever they are comfortable with
```
Comment (gdkchan) [inline README.md:127]: `allow` rather than `help`.

File: README.md (hunk 2 above)
Comment (gdkchan) [reply README.md:126]: I guess you're talking about PC parts rather than Switch hardware here? If so you should clarify.

File: README.md [hunk 4]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
+* Web hosting and infrastructure 
+ 
+All the funds acquired through our Patreon is considered a donation to support our work, and we will not do patron only releases. We may post exclusive progress reports, or early access to tools that can be used with the emulator.
```
Comment (gdkchan) [inline README.md:130]: `are` rather than `is`?

Comment (gdkchan) [reply README.md:130]: > or early access to tools that can be used with the emulator

Remove this as we are not doing that right now.

Comment (gdkchan) [reply README.md:130]: I'd also remove
 > , and we will not do patron only releases

File: README.md [hunk 5]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
-The Ryujinx.Audio project is licensed under the terms of the LGPLv3 license.
+This software is licensed under the terms of the <a href="https://github.com/Ryujinx/Ryujinx/blob/master/LICENSE.txt" target="_blank">MIT license.</a></i><br /> 
+The Ryujinx.Audio project is licensed under the terms of the <a href="https://github.com/Ryujinx/Ryujinx/blob/master/Ryujinx.Audio/LICENSE.txt
+" target="_blank">LGPLv3 license.</a></i><br />
 This project makes use of code authored by the libvpx project, licensed under BSD and the ffmpeg project, licensed under LGPLv3.
 See [LICENSE.txt](LICENSE.txt) and [THIRDPARTY.md](Ryujinx/THIRDPARTY.md) for more details.
- 
 ## Credits
 
+- [LibHac](https://github.com/Thealexbarney/LibHac) is used for our file-system
```
Comment (gdkchan) [inline README.md:141]:
```suggestion
- [LibHac](https://github.com/Thealexbarney/LibHac) is used for our file-system implementation.
```

File: README.md [hunk 6]
```diff
@@ -1,58 +1,83 @@
...
-    <a href="https://discord.gg/Ryujinx">
-        <img src="https://img.shields.io/discord/410208534861447168.svg">
-    </a>
+<h1 align="center">
+  <br>
+  <a href="https://ryujinx.org/"><img src="https://i.imgur.com/WcCj6Rt.png" alt="Ryujinx" width="150"></a>
+  <br>
+  <b>Ryujinx</b>
+  <br>
+  <sub><sup><b>(REE-YOU-JI-NX)</b></sup></sub>
```
Comment (gdkchan) [inline README.md:8]: I don't think including the spelling here is necessary. Does anyone even spell it like that?

**Comment by Shahil-Ayato** [comment]
Ready for review.

File: README.md [hunk 7]
```diff
@@ -1,58 +1,83 @@
...
 
+For our Local Wireless and LAN builds See our [Multiplayer: Local Play/Local Wireless Guide
+](https://github.com/Ryujinx/Ryujinx/wiki/Multiplayer-(LDN-Local-Wireless)-Guide)
+
 ## Latest build
 
 These builds are compiled automatically for each commit on the master branch. While we strive to ensure optimal stability and performance prior to pushing an update, our automated builds **may be unstable or completely broken.**
 
-The latest automatic build for Windows, macOS, and Linux can be found on the [Official Website](https://ryujinx.org/download).
+If you want to see what's changed in an update you can visit our [Changelog](https://github.com/Ryujinx/Ryujinx/wiki/Changelog).
```
Comment (LukeWarnut) [inline README.md:57]:
```suggestion
If you want to see what's changed in an update, you can visit our [Changelog](https://github.com/Ryujinx/Ryujinx/wiki/Changelog).
```

File: README.md [hunk 8]
```diff
@@ -1,58 +1,83 @@
...
+Don't hesitate to open a new issue if a game isn't already on there!
+                  
 ## Usage
 
-To run this emulator, we recommend that your PC have at least 8GB of RAM; less than this amount can result in unpredictable behavior and may cause crashes or unacceptable performance.
+To run this emulator, we recommend that your PC has at least 8GB of RAM; less than this amount can result in unpredictable behavior, crashes or unacceptable performance.
 
 See our [Setup & Configuration Guide](https://github.com/Ryujinx/Ryujinx/wiki/Ryujinx-Setup-&-Configuration-Guide) on how to set up the emulator.
 
+For our Local Wireless and LAN builds See our [Multiplayer: Local Play/Local Wireless Guide
```
Comment (LukeWarnut) [inline README.md:50]:
```suggestion
For our Local Wireless and LAN builds, see our [Multiplayer: Local Play/Local Wireless Guide
```

File: README.md [hunk 9]
```diff
@@ -1,58 +1,83 @@
...
-Download the ZIP Tarball. Then extract it to a directory of your choice.
+### Step 2
+Either use `git clone https://github.com/Ryujinx/Ryujinx` to clone the repository or use the `Code>Download zip` button to get the files
 
-**Step three:**
+### Step 3
 
-Build the App using a Command prompt in the project directory. You can quickly access it by holding shift in explorer (in the Ryujinx directory) then right clicking, and typing the following command:  
-Run `dotnet build -c Release` inside the Ryujinx project folder to build Ryujinx binaries.
+To build Ryujinx open the Command prompt in the project directory. You can quickly access it by holding shift in the file explorer(while inside the Ryujinx directory) then right clicking and selecting ``, then type the following command:
```
Comment (LukeWarnut) [inline README.md:74]:
```suggestion
To build Ryujinx, open a command prompt inside the project directory. You can quickly access it on Windows by holding shift in File Explorer, then right clicking and selecting `Open command window here`. Then type the following command:
```

**Review by LukeWarnut — CHANGES_REQUESTED** [review]

**Comment by Shahil-Ayato** [comment]
Feedback all addressed, Ready for review

File: README.md [hunk 10]
```diff
@@ -1,58 +1,83 @@
...
-After the installation of the .NET SDK is done; go ahead and copy the Clone link from GitHub from here (via Clone or Download --> Copy HTTPS Link. You can Git Clone the repo by using Git Bash or Git CMD.
+If you wish to build the emulator yourself, follow these steps:
 
-**(Variant two):**
+### Step 1
+Install the X64 version of [.NET 6.0 (or higher) SDK](https://dotnet.microsoft.com/download/dotnet/6.0).
 
-Download the ZIP Tarball. Then extract it to a directory of your choice.
+### Step 2
+Either use `git clone https://github.com/Ryujinx/Ryujinx` to clone the repository or use the `Code>Download zip` button to get the files
```
Comment (choppymaster) [inline README.md:70]:
```suggestion
Either use `git clone https://github.com/Ryujinx/Ryujinx` to clone the repository or use the `Download zip` button to get the files
```

Comment (Shahil-Ayato) [reply README.md:70]: Imo I don’t like removing `code` as not everyone would know/see where the download button is, so I’m leaving that change there to make it more streamlined for newer users.

**Review by choppymaster — COMMENTED** [review]
Some nits.

File: README.md [hunk 11]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
+</a>
 
-If you have contributions, need support, have suggestions, or just want to get in touch with the team, join our [Discord server](https://discord.gg/Ryujinx)!
-
-If you'd like to donate, please take a look at our [Patreon](https://www.patreon.com/ryujinx).
+All the developers working on the project do so on their free time, but the project has several expenses:
+* Hackable Nintendo Switch consoles to reverse-engineer the hardware
+* Additional hardware to test and fix any bugs certain vendors (AMD, Nvidia, and Intel) may have 
+* Software licenses to allow developers use whatever they are comfortable with
+* Web hosting and infrastructure
```
Comment (choppymaster) [inline README.md:128]:
```suggestion
* Web hosting and infrastructure maintaining
```

File: README.md [hunk 12]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
+<a href="https://www.patreon.com/ryujinx">
+    <img src="https://images.squarespace-cdn.com/content/v1/560c1d39e4b0b4fae0c9cf2a/1567548955044-WVD994WZP76EWF15T0L3/Patreon+Button.png?format=500w" width="150">
+</a>
 
-If you have contributions, need support, have suggestions, or just want to get in touch with the team, join our [Discord server](https://discord.gg/Ryujinx)!
-
-If you'd like to donate, please take a look at our [Patreon](https://www.patreon.com/ryujinx).
+All the developers working on the project do so on their free time, but the project has several expenses:
+* Hackable Nintendo Switch consoles to reverse-engineer the hardware
+* Additional hardware to test and fix any bugs certain vendors (AMD, Nvidia, and Intel) may have
```
Comment (choppymaster) [inline README.md:126]:
```suggestion
* Additional hardware to test (eg. GPUS to fix graphical bugs, other accessories to add support for, etc.) 
```

File: README.md [hunk 13]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
-If you have contributions, need support, have suggestions, or just want to get in touch with the team, join our [Discord server](https://discord.gg/Ryujinx)!
-
-If you'd like to donate, please take a look at our [Patreon](https://www.patreon.com/ryujinx).
+All the developers working on the project do so on their free time, but the project has several expenses:
+* Hackable Nintendo Switch consoles to reverse-engineer the hardware
+* Additional hardware to test (eg. GPUS to fix graphical bugs, other accessories to add support for, etc.) 
+* Software licenses to allow developers use whatever they are comfortable with
+* Web hosting and infrastructure maintaining
+ 
+All the funds acquired through our Patreon are considered a donation to support our work. Patrons get 1 week of exclusive early access to our progress reports and access to developer interviews.
```
Comment (choppymaster) [inline README.md:130]:
```suggestion
All the funds acquired through our Patreon are considered a donation to support our work. Patrons will get 1 week of exclusive early access to our progress reports and access to developer interviews.
```

File: README.md [hunk 14]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
 
 ## License
 
-This software is licensed under the terms of the MIT license.
-The Ryujinx.Audio project is licensed under the terms of the LGPLv3 license.
+This software is licensed under the terms of the <a href="https://github.com/Ryujinx/Ryujinx/blob/master/LICENSE.txt" target="_blank">MIT license.</a></i><br /> 
+The Ryujinx.Audio project is licensed under the terms of the <a href="https://github.com/Ryujinx/Ryujinx/blob/master/Ryujinx.Audio/LICENSE.txt
```
Comment (choppymaster) [inline README.md:135]:
```suggestion
The Ryujinx.Audio project is licensed under the terms of <a href="https://github.com/Ryujinx/Ryujinx/blob/master/Ryujinx.Audio/LICENSE.txt
```

File: README.md [hunk 15]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
 
-## Compatibility
-
-You can check out the compatibility list [here](https://github.com/Ryujinx/Ryujinx-Games-List/issues). Anyone is free to submit an updated test on an existing game entry; simply follow the new issue template and testing guidelines, and post as a reply to the applicable game issue.
 
-Don't hesitate to open a new issue if a game isn't already on there!
+## Contact
 
-## Help
+If you are having problems launching homebrew or a particular game marked status-playable or status-ingame in our compatibility list, or if you need help with setting up Ryujinx we recommend you ask questions in the #support channel of our [Discord server](https://discord.gg/Ryujinx) or take a look at our <a href="https://github.com/Ryujinx/Ryujinx/wiki/Frequently-Asked-Questions" target="_blank">FAQ.</a></i><br />  If you have contributions, have suggestions, or just want to get in touch with the team join our [Discord server](https://discord.gg/Ryujinx).
```
Comment (choppymaster) [inline README.md:113]:
```suggestion
If you are having problems launching homebrew or a particular game marked status-playable or status-ingame in our compatibility list, or if you need help with setting up Ryujinx we recommend you ask questions in the #support channel of our [Discord server](https://discord.gg/Ryujinx) or take a look at our <a href="https://github.com/Ryujinx/Ryujinx/wiki/Frequently-Asked-Questions" target="_blank">FAQ.</a></i><br />  If you have contributions, have suggestions, or just want to get in touch with the team, join our [Discord server](https://discord.gg/Ryujinx).
```

File: README.md [hunk 16]
```diff
@@ -1,58 +1,83 @@
...
-After the installation of the .NET SDK is done; go ahead and copy the Clone link from GitHub from here (via Clone or Download --> Copy HTTPS Link. You can Git Clone the repo by using Git Bash or Git CMD.
+If you wish to build the emulator yourself, follow these steps:
 
-**(Variant two):**
+### Step 1
+Install the X64 version of [.NET 6.0 (or higher) SDK](https://dotnet.microsoft.com/download/dotnet/6.0).
 
-Download the ZIP Tarball. Then extract it to a directory of your choice.
+### Step 2
+Either use `git clone https://github.com/Ryujinx/Ryujinx` to clone the repository or use the `Code>Download zip` button to get the files.
```
Comment (choppymaster) [inline README.md:70]:
```suggestion
Either use `git clone https://github.com/Ryujinx/Ryujinx` on the command line to clone the repository or use Code --> Download zip button to get the files.
```

File: README.md (hunk 14 above)
Comment (Shahil-Ayato) [reply README.md:135]: Imo removing that is unnecessary

File: README.md [hunk 17]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
+* Software licenses to allow developers use whatever they are comfortable with
+* Web hosting and infrastructure maintaining
+ 
+All the funds acquired through our Patreon are considered a donation to support our work. Patrons get 1 week of exclusive early access to our progress reports and access to developer interviews.
 
 ## License
 
-This software is licensed under the terms of the MIT license.
-The Ryujinx.Audio project is licensed under the terms of the LGPLv3 license.
+This software is licensed under the terms of the <a href="https://github.com/Ryujinx/Ryujinx/blob/master/LICENSE.txt" target="_blank">MIT license.</a></i><br />
```
[the remark this answers is not in the document]
Comment (Shahil-Ayato) [reply README.md:134]: Ditto

File: README.md [hunk 18]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
+    <img src="https://images.squarespace-cdn.com/content/v1/560c1d39e4b0b4fae0c9cf2a/1567548955044-WVD994WZP76EWF15T0L3/Patreon+Button.png?format=500w" width="150">
+</a>
 
-If you have contributions, need support, have suggestions, or just want to get in touch with the team, join our [Discord server](https://discord.gg/Ryujinx)!
-
-If you'd like to donate, please take a look at our [Patreon](https://www.patreon.com/ryujinx).
+All the developers working on the project do so on their free time, but the project has several expenses:
+* Hackable Nintendo Switch consoles to reverse-engineer the hardware
+* Additional hardware to test (eg. GPUS to fix graphical bugs, other accessories to add support for, etc.) 
+* Software licenses to allow developers use whatever they are comfortable with
```
Comment (choppymaster) [inline README.md:127]: Ditto here

Comment (choppymaster) [reply README.md:127]: Change to `Software licences (Visual Studio, etc)` etc, up to you.

Comment (Shahil-Ayato) [reply README.md:127]: Imo not needed.

Comment (EmulationFanatic) [inline README.md:127]: Development on this emulator takes place during contributors' free time. However, there are some inherent costs:
* Hackable Nintendo Switch consoles to reverse-engineer the hardware
* Additional computer hardware for testing purposes (e.g. GPUs to diagnose graphical bugs, etc.)
* Licenses for various software development tools

File: README.md [hunk 19]
```diff
@@ -82,31 +107,36 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
+* Web hosting and infrastructure maintaining
+ 
+All the funds acquired through our Patreon are considered a donation to support our work. Patrons will get 1 week of exclusive early access to our progress reports and access to developer interviews.
```
Comment (EmulationFanatic) [inline README.md:130]: * Web hosting and infrastructure maintenance

All funds received through Patreon are considered a donation to support the project. Patrons receive early access to progress reports and exclusive access to developer interviews.

File: README.md [hunk 20]
```diff
@@ -1,58 +1,83 @@
...
 </p>
 
 <h5 align="center">
-    As of September 2021, Ryujinx has been tested on nearly 3,400 titles: ~3,000 boot past menus and into gameplay, with approximately 2,400 of those being considered playable. See the compatibility list <a href="https://github.com/Ryujinx/Ryujinx-Games-List/issues" target="_blank">here</a>.
+  
 </h5>
 
+## Compatibility
+
+As of September 2021, Ryujinx has been tested on nearly 3,400 titles: ~3,000 boot past menus and into gameplay, with approximately 2,400 of those being considered playable.
```
Comment (EmulationFanatic) [inline README.md:39]: As of January 2022, Ryujinx has been tested on approximately 3,500 titles; over 3,200 boot past menus and into gameplay, with roughly 2,500 of those being considered playable.

**Review by EmulationFanatic — COMMENTED** [review]
Added my suggestions.

**Comment by Shahil-Ayato** [comment]
@EmulationFanatic great to see you again and I greatly appreciate your input! All feedback has been addressed, ready for review.

File: README.md [hunk 21]
```diff
@@ -82,31 +105,35 @@ Ryujinx system files are stored in the `Ryujinx` folder. This folder is located
...
+<a href="https://www.patreon.com/ryujinx">
+    <img src="https://images.squarespace-cdn.com/content/v1/560c1d39e4b0b4fae0c9cf2a/1567548955044-WVD994WZP76EWF15T0L3/Patreon+Button.png?format=500w" width="150">
+</a>
 
-If you'd like to donate, please take a look at our [Patreon](https://www.patreon.com/ryujinx).
+All the developers working on the project do so on their free time, but the project has several expenses:
+* Hackable Nintendo Switch consoles to reverse-engineer the hardware
+* Additional computer hardware for testing purposes (e.g. GPUs to diagnose graphical bugs, etc.) 
+* Licenses for various software development tools (e.g. Jetbrains, LDN servers, IDA)
+* Web hosting and infrastructure maintinance
```
Comment (EmulationFanatic) [inline README.md:125]: Typo.
"maintenance" is the proper spelling.

Comment (Shahil-Ayato) [reply README.md:125]: Agh don’t know how I let that one slip I’ll fix it asap

**Comment by Shahil-Ayato** [comment]
Feedback all addressed, ready for review/merge

**Pull request merged**